bootstrap/bootstrap-default.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Add hostname
+echo "127.0.0.1 $HOSTNAME" >>/etc/hosts
+
+# Make sure instance is updated with latest security fixes
```

Comment:
Let's remove commented code, and we will open an issue to get this done in a subshell (or something like that 😅 )